host/OEHOSTMR/README.md

```diff
@@ -0,0 +1,3 @@
+The `liboehostmr` is a subset of liboehost, and it's main functionality is to get the enclave's measurement. Following other host libraries' naming convention, this library starts with liboehost, and `mr` is short for `measurement report`.
+
+To reuse source code, a macro `OEHOSTMR` is used that some sentences refering to libsgx_enclave_common are stripped off. Thus this library has no dependency on libsgx_enclave_common(also libsgx_dcap_ql).
```

Comment:
```suggestion
The `liboehostmr` library reuses the source code from `liboehost` by using the preprocessor definition `OEHOSTMR`. When specified, `OEHOSTMR` removes any code references to libsgx_enclave_common or libsgx_dcap_ql so that `liboehostmr` has no dependencies on those libraries.
```

Reply:
It looks like there's a merge conflict and both upper case and lower case versions of the file exist in this PR now?

Reply:
Thanks for rewriting the whole paragraph! Yes I forget to use `git mv` but `mv` only, now it is fixed.